src/auth_scheme/private_token.ts

```diff
@@ -407,9 +529,18 @@ export class AuthorizationHeader {
             attrKey = attrKey.trim();
             attrValue = attrValue.trim();
 
-            if (attrKey === 'token') {
-                const tokenEnc = base64url.parse(attrValue);
-                ppToken = Token.deserialize(tokenTypeEntry, tokenEnc);
+            switch (attrKey) {
+                case 'token': {
+                    const tokenEnc = base64url.parse(attrValue);
+                    ppToken = Token.deserialize(tokenTypeEntry, tokenEnc);
+                    break;
+                }
+                case 'extensions': {
+                    if (tokenTypeEntry.publicMetadata) {
+                        const extEnc = base64url.parse(attrValue);
+                        extentions = Extensions.deserialize(extEnc);
```

Comment:
does JS need break here?

```suggestion
                        extentions = Extensions.deserialize(extEnc);
                        break;
```

Reply:
it does